### PR TITLE
user favorites & uploaded-resources func added

### PR DESCRIPTION
**Pull request by abubaker47**
Users should be able to see there favorite and uploaded-resources on there profile. Issue https://github.com/ddlibrary/website/issues/158 Fixed.

### CODE REVIEW

**Review by uyscuti-wiki — CHANGES_REQUESTED** [review]
This looks good. Few comments inline.

File: resources/views/users/favorites.blade.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                <tr>
+                    <th>#</th>
+                    <th>@lang('Title')</th>
+                    <th>@lang('Actions')</th>
+                </tr>
+            </thead>
+            <tbody>
+                @foreach ($resources as $resource)
+                    <tr>
+                        <td>{{ ++$c }}</td>
```
Comment (uyscuti-wiki) [inline resources/views/users/favorites.blade.php:20]: `$c` used before being defined.

File: resources/views/users/uploaded-resources.blade.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                <tr>
+                    <th>#</th>
+                    <th>@lang('Title')</th>
+                    <th>@lang('Actions')</th>
+                </tr>
+            </thead>
+            <tbody>
+                @foreach ($resources as $resource)
+                    <tr>
+                        <td>{{ ++$c }}</td>
```
Comment (uyscuti-wiki) [inline resources/views/users/uploaded-resources.blade.php:20]: `$c` used before being defined.

File: resources/views/users/favorites.blade.php [hunk 3]
```diff
@@ -0,0 +1,31 @@
+@extends('layouts.main')
+@section('content')
+<section class="general-content">
+    <header>
+        <h1>@lang('Users Details for') <strong>{{ $user->username }}</strong></h1>
```
[the remark this answers is not in the document]
Comment (abubaker47) [reply resources/views/users/favorites.blade.php:5]: Thanks for addressing this, I have resolved it.

File: resources/views/users/favorites.blade.php [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+                        <td><a href="{{ URL::to('resource/' . $resource->id) }}">@lang('View')</a></td>
+                    </tr>
+                @endforeach
+            </tbody>
+        </table>
+    </article>
+</section>
+<!-- /.container-fluid-->
+<!-- /.content-wrapper-->
+@endsection
```
[the remark this answers is not in the document]
Comment (abubaker47) [reply resources/views/users/favorites.blade.php]: Resolved.

File: resources/views/users/uploaded-resources.blade.php [hunk 5]
```diff
@@ -0,0 +1,31 @@
+@extends('layouts.main')
+@section('content')
+<section class="general-content">
+    <header>
+        <h1>@lang('Users Details for') <strong>{{ $user->username }}</strong></h1>
```
[the remark this answers is not in the document]
Comment (abubaker47) [reply resources/views/users/uploaded-resources.blade.php:5]: Resolved.

File: resources/views/users/uploaded-resources.blade.php [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+                        <td><a href="{{ URL::to('resource/' . $resource->id) }}">@lang('View')</a></td>
+                    </tr>
+                @endforeach
+            </tbody>
+        </table>
+    </article>
+</section>
+<!-- /.container-fluid-->
+<!-- /.content-wrapper-->
+@endsection
```
[the remark this answers is not in the document]
Comment (abubaker47) [reply resources/views/users/uploaded-resources.blade.php]: Resolved.

File: resources/views/users/user_nav.blade.php [hunk 7]
```diff
@@ -0,0 +1,7 @@
+<nav class="user-nav">
+<a href="{{ URL::to('user/profile') }}" title="@lang('Personal Information')" class="{{ ($page == 'profile') ? 'active' : '' }}">@lang('Personal Information')</a>
+    <a href="{{ URL::to('user/favorites') }}" title="@lang('Favorites')" class="{{ ($page == 'favorites') ? 'active' : '' }}">@lang('Favorites')</a>
+    <a href="{{ URL::to('user/uploaded-resources') }}" title="@lang('Uploaded Resources')" class="{{ ($page == 'uploaded-resources') ? 'active' : '' }}">@lang('Uploaded Resources')</a>
+
+<hr class="hr-class">
+</nav>
```
[the remark this answers is not in the document]
Comment (abubaker47) [reply resources/views/users/user_nav.blade.php]: Resolved.

File: resources/views/users/favorites.blade.php (hunk 1 above)
Comment (abubaker47) [reply resources/views/users/favorites.blade.php:21]: Resolved.

File: resources/views/users/uploaded-resources.blade.php (hunk 2 above)
Comment (abubaker47) [reply resources/views/users/uploaded-resources.blade.php:21]: Resolved.

**Pull request merged**